RadarSDK/RadarAPIClient.m

```diff
@@ -90,22 +105,18 @@ - (void)getConfig {
                                 sleep:NO
                            logPayload:YES
                     completionHandler:^(RadarStatus status, NSDictionary *_Nullable res) {
-                        if (!res) {
-                            return;
-                        }
+        if (!res) {
+            return;
+        }
```

Comment:
looks like the merge might have wiped out 

```
[Radar flushLogs];
```

Reply:
Done.